frame/support/src/traits/hooks.rs

```diff
@@ -22,6 +22,7 @@ use impl_trait_for_tuples::impl_for_tuples;
 use sp_runtime::traits::AtLeast32BitUnsigned;
 use sp_std::prelude::*;
 
+#[cfg(test)]
```

Comment:
What you want to achieve here? That this is also enabled when we are compiling the tests?

Reply:
This has to be enabled only when it's both `test` and `try-runtime` and is exactly what it currently does.

Reply:
To give some context:
```
#[cfg(feature = "try-runtime")]
use codec::{Decode, Encode};
```
Will give an unused import error, because this import is only used within tests, so when we are building with try-runtime - there's a warning emitted and build fails.

Reply:
To give a bit more context `cargo remote -- test --features=try-runtime` this is the only time this import has to be enabled.

Reply:
`#[cfg(all(feature = "try-runtime", test))]`

Reply:
yeah, I was looking for that :)